### PR TITLE
test: dashboard test

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [x] E2E tests  
- [ ] Docs  
- [ ] CI/CD  

Peek 2022-04-21 10-02

On errors: 

Screenshot from 2022-04-21 09-36-37

### CODE REVIEW

File: packages/e2e/cypress/integration/dashboard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        ['chat.lightdash.com', 'www.loom.com'].forEach((url) => {
+            cy.intercept(
+                {
+                    hostname: url,
+                },
+                (req) => {
+                    req.destroy();
+                },
+            );
+        });
```
Comment (ZeRego) [inline packages/e2e/cypress/integration/dashboard.spec.ts:19]: This is great! 😃 
We could have this for all the tests if we define it in the `cypress/support/index.js` ( [blog post explaining](https://ashley-sinner.medium.com/cypress-index-support-file-write-once-run-everywhere-4106c297a1aa) )

**Review by ZeRego — CHANGES_REQUESTED** [review]
Thanks for writing more e2e tests 🙌  🥳 

Left a couple of comments + I think we should rebase this branch since we already merged the reset-db script to main.

**Comment by rephus** [comment]
I had to migrate and seed again on render to get it to work... it was not a problem of my e2e test . In fact, the e2e test was catching the issue !!!

**Comment by rephus** [comment]
Finally !!! 
image

**Review by ZeRego — APPROVED** [review]

**Pull request merged**